### ISSUE TITLE
Incorrect using of AttributionText in SPDX output

### ISSUE THREAD

**Issue by knqyf263** [aquasecurity/trivy#7756]
## Description
We should use `annotations` for scan metadata instead of `attributionTexts`.

```
"annotations" : [ {
    "annotationDate" : "2024-10-29T18:30:22Z",
    "annotationType" : "OTHER",
    "annotator" : "Tool: Trivy ()",
    "comment" : "Class: lang-pkgs"
  },
  {
    "annotationDate" : "2024-10-29T18:30:22Z",
    "annotationType" : "OTHER",
    "annotator" : "Tool: Trivy ()",
    "comment" : "Type: npm"
  },
```

https://github.com/aquasecurity/trivy/blob/c8c14d36245623019f29d258f813d2325f7490f7/pkg/sbom/spdx/marshal.go#L369-L378

### Discussed in https://github.com/aquasecurity/trivy/discussions/7715

**Comment by knqyf263** [comment]
@DmitriyLewen Do you think we can fix it for v0.57.0? If this task takes some time, I'll put it into v0.58.0.

**Comment by DmitriyLewen** [comment]
I will take a look today and write to you

**Comment by DmitriyLewen** [comment]
@knqyf263  while I was checking - I created #7811 😄

**Comment by DmitriyLewen** [comment]
fixed in #7811
